pyirf/utils.py

```diff
@@ -158,5 +168,5 @@ def check_table(table, required_columns=None, required_units=None):
                 raise MissingColumns(col)
 
             unit = table[col].unit
-            if not expected.is_equivalent(unit):
+            if not unit or not expected.is_equivalent(unit):
```

Comment:
The `not unit`-case should be tested in `tests/test_utils`., e.g. by passing a `Table` instead of a `QTable`.

Reply:
Done